package.json

```diff
@@ -24,13 +24,15 @@
     "start": "pm2-runtime start pm2.config.js",
     "heroku-postbuild": "echo 'Skipping build'",
     "build": "node scripts/prod.js",
-    "clean": "rm -rf ./packages/client/__generated__ ./queryMap.json ./packages/server/graphql/public/schema.graphql ./packages/server/graphql/private/schema.graphql ./dev ./dist ./build",
+    "clean": "find . \\( -name node_modules -o -name dev -o -name dist -o -name build -o -name '__generated__' -o -name queryMap.json -o -name schema.graphql -o -name resolverTypes.ts \\) -prune | xargs rm -rf",
```

Comment:
an idea, what if instead of finding the files manually, we'd do `"mv .env .env.backup && git clean -fdx && cp .env.backup .env"`

1. rename .env to be something not ignored by git
2. run `git clean -fdx` which removes everything which is gitignored
3. restore .env

any thoughts?